.github/workflows/helium-unity.yml

```diff
@@ -73,7 +73,7 @@ jobs:
 
   tests:
     name: Unity Actions & Unit Testing
-    runs-on: cb-office-mac-intel-1
+    runs-on: macos-latest
```

Comment:
Is there a way to specify the Xcode version as well? In case we also need to declare this.

Reply:
This is not running anything on XCode, and most likely never will @cb-jpadilla . This is just for UPM Repo. Canary will still run on self-hosted.